### ISSUE TITLE
[Question]: while I'm training with RecurrentPPO the entropy_loss becomes positive. How is this possible?

### ISSUE THREAD

**Issue by LucaGiorcelli** [Stable-Baselines-Team/stable-baselines3-contrib#282]
### ❓ Question

During training with recurrentPPO, the entropy loss goes from -8(first logged value) to +2. It was still increasing but i stopped training. I'm using a custom environment with a continuous action space (-1,1).

I don't understand how this value can become positive, shouldn't it converge to zero?

The other metrics are converging to the correct values: the explained variance converged to 0.99 and the std is 0.2 but still decreasing. 

Unfortunately I cannot provide the environment.

These are the hyperparameters I'm currently using:
model = RecurrentPPO(
    policy='MlpLstmPolicy',
    env=env,
    batch_size=128,
    verbose=1,
    clip_range=0.2, 
    ent_coef=1e-4,
    learning_rate=1.5e-4,
    vf_coef=0.5,
    n_epochs=30,
    n_steps=12800,
    policy_kwargs=dict(net_arch=dict(pi=[128,128], vf=[128,128]), lstm_hidden_size=128, n_lstm_layers = 2),
    gamma=0.99,
)

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues) in the repo
- [x] I have read the [documentation](https://sb3-contrib.readthedocs.io/en/master/)
- [x] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [x] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.
